pkg/frontend/load.go

```diff
@@ -1737,6 +1776,9 @@ func writeBatchToStorage(handler *WriteBatchHandler, force bool) error {
 					case types.T_timestamp:
 						cols := vec.Col.([]types.Timestamp)
 						vec.Col = cols[:needLen]
+					case types.T_uuid:
+						cols := vec.Col.([]types.Uuid)
```

Comment:
Here, as well.   Actually why don't you change everything above?

Reply:
OK, I've changed all these